src/makers/heltec/antennas.ts

```diff
@@ -0,0 +1,28 @@
+import { MakerAntenna } from '../antennaMakerTypes'
+
+const HELTEC_HOTSPOT = {
+  name: 'Heltec Hostpot (all)',
```

Comment:
@cokes518 the `(all)` here seems a bit confusing. Maybe we change it to `(default)` or remove it?

Reply:
Sorry for my poor English, already updated this part.

Reply:
Thank you, looks good!